cellxgene_schema_cli/Makefile

```diff
@@ -1,5 +1,5 @@
-.PHONY: update-references
-update-references: gene-processing clean
+.PHONY: update-gene-references
```

Comment:
this is no longer a generic operation for ontology and genes alike. It is now just genes and should be labeled as such.